nodejs/eks/cluster.ts

```diff
@@ -1379,15 +1379,15 @@ export class Cluster extends pulumi.ComponentResource {
         // Check that AWS provider credential options are set for the kubeconfig
         // to use with the given auth method.
         if (opts?.provider && !args.providerCredentialOpts) {
-            throw new Error("providerCredentialOpts and an AWS provider instance must be set together");
+            throw new Error("It looks like you're using an explicit AWS provider. Please specify this provider in providerCredentialOpts. For more details, see: https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/eks/#ClusterOptions-providerCredentialOpts")
```

Comment:
Those are node.js docs - but this error will show up in all languages - right? Can we link to language-agnostic resource docs for this?

Reply:
ah this is a good point, I'll remove this for now until we figure out a way to make the link language agnostic.